src/index.html

```diff
@@ -10,10 +10,42 @@
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+      <div class="card-block" data-qa="card">
+        <div class="imac"><img class="block image" src="images/imac.jpeg" alt="card image" /></div>
+        <div class="about-block">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+        <div class="block-code">Код товара: 195434</div>
+        <div class="card rating stars">
+          <ul class="card block-stars">
+            <li class="block-star item">
+              <img class="stars" src="images/star-active.svg" alt="active rating" />
+            </li>
+            <li class="block-star item">
+              <img class="stars" src="images/star-active.svg" alt="active rating" />
+            </li>
+            <li class="block-star item">
+              <img class="stars" src="images/star-active.svg" alt="active rating" />
+            </li>
+            <li class="block-star item">
+              <img class="stars" src="images/star-active.svg" alt="active rating" />
+            </li>
+            <li class="block-star item">
+              <img class="stars" src="images/star.svg" alt="active rating" />
+            </li>
+          </ul>
+          <div class="points">Отзывов: 5</div>
+        </div>
+        <div class="card rating prices">
+          <p class="price">Цена:</p>
+          <p class="amount">69 999 грн</p>
```

Comment:
![image](https://user-images.githubusercontent.com/44294945/64256543-5ef2ec80-cf2c-11e9-96d1-b90321be3c63.png)

better to add this margin to the parent element `div`